pkg/protocompat/message_test.go

```diff
@@ -27,7 +27,7 @@ func TestClone(t *testing.T) {
 	clonedNamespace, casted := cloned.(*storage.NamespaceMetadata)
 	assert.True(t, casted)
 	clonedNamespace.Name = "Namespace AA"
-	assert.NotEqual(t, m1, cloned)
+	assert.False(t, Equal(m1, cloned))
```

Comment:
Can we use `protoassert.NotEqual` here?

Reply:
Yes but I think it's better to keep it as is and remove `protoassert` from this package (there are just 2 places in this file) to not introduce unnecessary coupling and maybe cyclic dependency in the future.